app/src/main/java/org/simple/clinic/home/overdue/OverdueEvent.kt

```diff
@@ -35,7 +35,11 @@ data class DownloadOverdueListClicked(
   override val analyticsName = "Overdue Screen:Download clicked"
 }
 
-object ShareOverdueListClicked : OverdueEvent() {
+data class ShareOverdueListClicked(
+    override var permission: Optional<RuntimePermissionResult> = Optional.empty(),
+    override val permissionString: String = Manifest.permission.WRITE_EXTERNAL_STORAGE,
+    override val permissionRequestCode: Int = 1
```

Comment:
Can we have a different permission request code for this, please? It can just be 2

Reply:
done